backend/src/main/java/org/example/app/task/service/TaskService.java

```diff
@@ -74,11 +78,11 @@ public class TaskService {
   @APIResponse(responseCode = "500", description = "Server unavailable or a server-side error occurred")
   public Response saveTask(@Valid TaskListEto taskList) {
 
-    Long taskListId = this.ucSaveTaskList.save(taskList);
-    if (taskList.getId() == null || taskList.getId() != taskListId) {
-      return Response.created(URI.create("/task/list/" + taskListId)).build();
+    TaskListEntity savedTaskList = this.ucSaveTaskList.save(taskList);
```

Comment:
Returning an Entity from a service is not good practice. We'll have a talk about this in the training